timesketch/lib/datastores/elastic.py

```diff
@@ -11,8 +11,9 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-"""ElasticSearch datastore."""
+"""Elasticsearch datastore."""
```

Comment:
nit: In all the comments, Elasticsearch is used. But this lib has class ElasticsearchDataStore.